Fix misleading path when rendering partial from component itself (#511)
octobercms/docs: plugin-components.md

<a name="render-partial-method"></a>
## Rendering partials from code

> You may programmatically render component partials inside the PHP code using the `renderPartial` method. This will check the component for the partial named `component-partial.htm` and return the result as a string. The second parameter is used for passing view variables. The same [path resolution logic](#component-partials) operates when you render a component partial from the backend, use the `@` prefix to refer to partials within the component itself.

Reviewer: ```suggestion
You may programmatically render component partials inside the PHP code using the `renderPartial` method. This will check the component for the partial named `component-partial.htm` and return the result as a string. The second parameter is used for passing view variables. The same [path resolution logic](#component-partials) applies when you render a component partial in PHP as it does with Twig; use the `@` prefix to refer to partials within the component itself.
```

> You may programmatically render component partials inside the PHP code using the `renderPartial` method. This will check the component for the partial named `component-partial.htm` and return the result as a string. The second parameter is used for passing view variables. The same [frontend syntax rules](#component-partials) operates when you render a component partial from the backend, use the `@` prefix to refer the partial path from the component itself.

Reviewer: ```suggestion
You may programmatically render component partials inside the PHP code using the `renderPartial` method. This will check the component for the partial named `component-partial.htm` and return the result as a string. The second parameter is used for passing view variables. The same [path resolution logic](#component-partials) operates when you render a component partial from the backend, use the `@` prefix to refer to partials within the component itself.
```
